### DISCUSSION TITLE
My Github account seems to have been hacked

### DISCUSSION THREAD

**Discussion by Y5neKO** [community/community#103073] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Someone logged into my account seven hours ago and created a repository.
I checked the security log just now. Has anyone encountered a similar situation?
I'm not sure if the account was hacked or it was a github robot or something like that.
![image](https://github.com/community/community/assets/41314905/c1c3dc81-5f36-42f4-8f84-8d4caa9e78dd)

**Comment by Y5neKO** [comment]
I just discovered that my account starred a warehouse four hours ago, but I have not performed this operation.
![image](https://github.com/community/community/assets/41314905/09e3a6b1-9ff2-4369-a13e-74123714df17)
![image](https://github.com/community/community/assets/41314905/961c00ff-d148-48fc-ae30-85a15610cc40)

**Comment by Y5neKO** [comment]
All stars in this repository are compromised users, because everyone automatically created a repository of "{user}1"
![image](https://github.com/community/community/assets/41314905/be8d58d7-ed62-470e-a459-bd449eeb5f24)
![image](https://github.com/community/community/assets/41314905/aeba6e52-a032-4b2e-bfd8-e71f058710e6)
![image](https://github.com/community/community/assets/41314905/614e0fbb-4ffb-4fa7-aa2c-b343d45e8b6b)
![image](https://github.com/community/community/assets/41314905/977c6989-26c0-4317-929a-56e44fe4033e)

**Reply by Pudv95** [reply]
hey i faced the same thing
what should i do next 
do you know why it happened or anything else....?

**Comment by gil-son** [comment]
So, try to contact [Github Support](https://support.github.com/)

and:

- Change Your Password
- If you haven't already, enable two-factor authentication on your GitHub account

**Reply by Y5neKO** [reply]
ok,thank you~

**Comment by ghostinhershell** [comment]
Hi @Y5neKO, 👋🏾 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)
- You can also open a ticket on our [Support page](https://support.github.com/contact).

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
